docs/source/pipelines.rst

```diff
@@ -4,194 +4,321 @@
-or perform an MLOps task, such as developing an accurate regression model for data within a
-particular domain or distributing a model to perform scalable batch inference. Pipelines
-structure their code, configurations, and tests as git repositories with a standardized
-directory hierarchy, as discussed in :ref:`Pipeline Repositories
-<pipeline-repositories-key-concept>`. MLflow Pipelines includes predefined pipeline repositories
-for common ML problems and MLOps tasks called :ref:`Pipeline Templates <pipeline-templates>`.
-MLflow Pipelines also provides :py:class:`APIs <mlflow.pipelines.Pipeline>` and a :ref:`CLI <cli>`
-for running pipelines and inspecting their results.
+.. _step-cards-key-concept:
 
-.. code-section::
+- **Step Cards**: Step Cards display :ref:`Pipeline <pipelines-key-concept>` results, including
+  dataset profiles, model performance & explainability plots, overviews of the best model
+  parameters found during tuning, and more. Step Cards and their corresponding dataset and model
+  information are also logged to MLflow Tracking.
 
-    .. code-block:: python
-      :caption: Example API and CLI workflows for running the :ref:`MLflow Regression Pipeline
-                <mlflow-regression-pipeline>` and inspecting results. Note that pipelines
-                must be run from within their corresponding git repositories.
+Usage
+-----
 
-      import os
-      from mlflow.pipelines import Pipeline
-      from mlflow.pyfunc import PyFuncModel
+The general workflow for using MLflow Pipelines is as follows:
 
-      os.chdir("~/mlp-regression-template")
-      regression_pipeline = Pipeline(profile="local")
-      # Run the full pipeline
-      regression_pipeline.run()
-      # Inspect the model training results
-      regression_pipeline.inspect(step="train")
-      # Load the trained model
-      regression_model_pipeline: PyFuncModel = regression_pipeline.get_artifact("model")
+1. Clone a :ref:`Pipeline Template <pipeline-templates>` git repository corresponding to the ML
+   problem that you want to solve or the MLOps task that you want to perform. View the template's
+   README file for information about the :ref:`Pipeline Steps <steps-key-concept>` that it defines
+   and the results that it produces.
 
     .. code-block:: sh
+      :caption: An example of cloning the |MLflow Pipelines Regression Template|
 
       git clone https://github.com/mlflow/mlp-regression-template
-      cd mlp-regression-template
-      # Run the full pipeline
-      mlflow pipelines run --profile local
-      # Inspect the model training results
-      mlflow pipelines inspect --step train --profile local
-      # Inspect the resulting model performance evaluations
-      mlflow pipelines inspect --step evaluate --profile local
 
-.. _pipeline-repositories-key-concept:
+    .. note::
+      On Databricks, we recommend cloning the :ref:`Pipeline Template <pipeline-templates>` git
+      repository using |Databricks Repos|.
```

Comment:
Isn't repos their only option?

Reply:
You could technically do `%sh git clone ...`, which is what we do for integration tests. Since this section is preexisting and updates are owned by @jinzhang21 , I'm going to leave this as-is for now.

Reply:
Don't have all the context, but that seems a little contrived unless a lot of people are doing it that way. Could we be prescriptive instead? ie `On Databricks, clone the ... git repository using Databricks Repos`

Reply:
Don't have all the context, but that seems a little contrived unless a lot of people are doing it that way. Could we be prescriptive instead? ie `On Databricks, clone the ... git repository using Databricks Repos`

Reply:
Makes sense to me. I'll update this section later.